auth-web/src/util/constants.ts

```diff
@@ -405,7 +405,13 @@ export enum NrRequestTypeStrings {
 
 export enum BusinessState {
     ACTIVE = 'Active',
-    DRAFT = 'Draft'
+    DRAFT = 'Draft',
+    AWAITING_REVIEW = 'Pending Staff Review',
+    CHANGE_REQUESTED = 'Change Requested',
+    REJECTED = 'Rejected',
+    HISTORICAL = 'Historical',
+    PENDING = 'Pending | Payment Incomplete',
+    PAID = 'Future Effective' // Future effective and paid
```

Comment:
To be clear, this state could mean:
- a future effective filing
- an immediate filing that the Filer hasn't processed yet (maybe "Paid | Pending"? ask Andy) (this could also be an overdue future effective filing)

In order to keep the strings here, I think you'll need a new business state here (PAID_FE?), and then you'll need to select between them in affiliations-factory.ts.

Also, you should handle APPROVED state the same as PAID (and Approved could also be FE).

Reply:
Sample future-effective, approved continuation application:

https://dev.business.bcregistry.gov.bc.ca/Tzg27dGHVo?filing_id=150492&accountid=2288

Reply:
Would it make sense to have a "getBusinessState" function in which to encapsulate the string lookup and special logic? Maybe we don't need the enum either if everything is in one place.

Idea:
```
function getBusinessState (status): string {
   switch (status) {
      case 'PAID': return (isFutureEffective) ? 'Future Effective' : 'Paid | Pending'
      case 'DRAFT': return 'Draft'
      ...
      default: return '' // or 'Unknown'?
   }
}
```

Reply:
Updated status:
- PAID (filer pending)
- APPROVED
- PAID_FE

I think may be better to keep the enums, because I used them in the AffliationAction file to determine actions for different statuses; and computed some statuses when fetch from api

Reply:
OK. I appreciate the effort to not make too many changes. Hopefully when this is ported to another UI, we can simplify the code somewhat.